conduit-hyper/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "conduit-hyper"
+version = "0.4.2"
+authors = ["Justin Geibel <jtgeibel@gmail.com>"]
+license = "MIT OR Apache-2.0"
+description = "Host a conduit based web application on a hyper server"
+repository = "https://github.com/jtgeibel/conduit-hyper"
```

Comment:
How about updating this URL and publishing a new version just in case? It'd be helpful when a crate seeker finds the crate.